src/test/unit/algorithms/CondProbTableTest.cpp

```diff
@@ -180,13 +183,13 @@ namespace nupic {
 
 
   //----------------------------------------------------------------------
-  void CondProbTableTest::testTable(const string& testName, CondProbTable& table, 
-                                    const vector<vector<Real> > & rows)
+  void testTable(const string& testName, CondProbTable& table, 
+                 const vector<vector<Real> > & rows)
   {
   
     // Test the numRows(), numCols() calls
-    TESTEQUAL(numRows(), table.numRows());
-    TESTEQUAL(numCols(), table.numColumns());
+    EXPECT_EQ(numRows(), table.numRows());
```

Comment:
@scottpurdy My aim in translating these tests is to not modify the behavior of the tests. In https://github.com/numenta/nupic.core/blob/master/src/nupic/test/Tester.hpp#L116-L117, TESTEQUAL is defined as EXPECT_EQ.